### PR TITLE
vote: v01dstar as TiKV reviewer

### PR DESCRIPTION
**Pull request by Connor1996**

### CODE REVIEW

**Comment by Connor1996** [comment]
@tonyxuqqi

File: teams/tikv/team.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
         "rleungx",
         "tier-cap",
         "tonyxuqqi",
+	"v01dstar",
```
Comment (iosmanthus) [inline teams/tikv/team.json:76]: The indent is wrong.

Comment (Connor1996) [reply teams/tikv/team.json]: fixed

**Review by BusyJay — APPROVED** [review]
Thanks for your contributions! @v01dstar

**Review by tonyxuqqi — APPROVED** [review]

**Review by iosmanthus — APPROVED** [review]

**Review by zhangjinpeng1987 — APPROVED** [review]
LGTM

**Pull request merged**